location.go

```diff
@@ -0,0 +1,52 @@
+package gsclient
+
+import (
+	"net/http"
+	"path"
+)
+
+//LocationList JSON struct of a list of locations
+type LocationList struct {
+	List map[string]LocationProperties `json:"locations"`
+}
+
+//Location JSON struct of a single location
+type Location struct {
+	Properties LocationProperties `json:"location"`
+}
+
+//LocationProperties JSON struct of properties of a location
+type LocationProperties struct {
+	Iata       string   `json:"iata"`
+	Status     string   `json:"status"`
+	Labels     []string `json:"labels"`
+	Name       string   `json:"name"`
+	ObjectUuid string   `json:"object_uuid"`
```

Comment:
I would prefer to fix all field names with UUID; for instance, ObjectUuid should be ObjectUUID. `golint` tool will help you on fixing the name convention.